### PR TITLE
Some clean up for mod/item.php

### PR DESCRIPTION
**Pull request by annando**
We now use some more central functions and the queries had been improved.

### CODE REVIEW

File: mod/item.php [hunk 1]
```diff
@@ -346,14 +346,14 @@ function item_post(App $a) {
 	if ($parent) {
 		if ($thr_parent_contact['network'] == NETWORK_OSTATUS) {
 			$contact = '@[url=' . $thr_parent_contact['url'] . ']' . $thr_parent_contact['nick'] . '[/url]';
-			if (!in_array($contact, $tags)) {
+			if (!stripos(implode($tags), '[url=' . $thr_parent_contact['url'] . ']')) {
```
Comment (MrPetovan) [inline mod/item.php]: Aren’t you supposed to provide a glue parameter?

Comment (annando) [reply mod/item.php]: Doesn't seem so: http://php.net/manual/en/function.implode.php

Comment (MrPetovan) [reply mod/item.php]: Interesting, I never used it like this.

**Comment by annando** [comment]
I tested, but now I looked into my PHP log:
```PHP Warning:  implode(): Argument must be an array in /var/www/virtual/pirati.ca/htdocs/mod/item.php on line 962```
So I will do a next change.

**Comment by annando** [comment]
I don't see warnings. So it should be okay.

**Pull request merged**